src/lib/is_array.js

```diff
@@ -8,15 +8,11 @@
 
 'use strict';
 
-/**
- * Return true for arrays, whether they're untyped or not.
- */
-
 // IE9 fallback
 var ab = (typeof ArrayBuffer === 'undefined' || !ArrayBuffer.isView) ?
     {isView: function() { return false; }} :
     ArrayBuffer;
 
-module.exports = function isArray(a) {
+exports.isArrayOrTypedArray = function(a) {
```

Comment:
perusing https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/isView - there's also `DataView` that passes but looks like it's a little too generic to be meaningful to us as an input. Should we be concerned about this?

Reply:
improved in f0395b5965b037af7d3ac24e04057ba532a35d1b